src/mixer/basetrackplayer.cpp

```diff
@@ -548,6 +557,13 @@ void BaseTrackPlayerImpl::slotCloneFromDeck(double d) {
     }
 }
 
+void BaseTrackPlayerImpl::slotCloneFromSampler(double d) {
+    int sampler = std::lround(d);
```

Comment:
Normally we just cast to an int in this case.
The round is also part in the deck version so I am unsure.
I think we should do in both cases static_cast to not create an exception.
@Holzhaus: Since you did the double vs int fight, what do you think?

Reply:
Yes, static_cast makes sense.

Reply:
alright, I'll apply that to all teh related slots here.